plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -113,6 +115,27 @@ public void testBasicTypes()
                 .execute(getQueryRunner(), trinoCreateAsSelect("test_basic_types"));
     }
 
+    @Test
+    public void testBit()
```

Comment:
Since these tests should be passing before the MySqlClient change, add them as preparatory commit.